crates/core/tedge_mapper/src/aws/mapper.rs

```diff
@@ -59,3 +73,30 @@ fn get_topic_filter(tedge_config: &TEdgeConfig) -> TopicFilter {
     }
     topics
 }
+
+fn built_in_bridge_rules(remote_client_id: &str) -> Result<BridgeConfig, anyhow::Error> {
+    let local_prefix = "aws/";
+    let device_id_prefix = format!("thinedge/{remote_client_id}/");
+    let things_prefix = format!("$aws/things/{remote_client_id}/");
+    let conn_check = format!("thinedge/devices/{remote_client_id}/test-connection");
+    let mut bridge = BridgeConfig::new();
+
+    // telemetry/command topics for use by the user
+    bridge.forward_from_local("td/#", local_prefix, device_id_prefix.clone())?;
+    bridge.forward_from_remote("cmd/#", local_prefix, device_id_prefix)?;
+
+    // topic to interact with the shadow of the device
+    bridge.forward_from_local("shadow/#", local_prefix, things_prefix.clone())?;
+    bridge.forward_from_remote("shadow/#", local_prefix, things_prefix)?;
```

Comment:
How do we ensure this doesn't lead to an infinite loop?

Reply:
We only have one client connected to the local broker, and we shouldn't receive messages that we originally sent. I'll attempt to add a test for this.